CodeStyleExercise/ColorsViewController.swift

```diff
@@ -67,28 +64,16 @@ class ColorsViewController : UIViewController, UICollectionViewDataSource, UITex
         }
     }
```

Comment:
This entire code block is a good place to use `guard` statements instead of nested ifs (pyramid of doom)